sktime/libs/fracdiff/__init__.py

```diff
@@ -0,0 +1,43 @@
+"""Fractional difference , a la "Advances in Financial Machine Learning" by M. Prado
+
+Unofficial fork of the ``fracdiff`` package, maintained in ``sktime``.
+
+sktime migration: 2024, July
+Version 0.9.1 release: 2023, Feb 4 (DinoBektesevic)
```

Comment:
this is not correct, is it? It should be the last releast on pypi and account name.
Migration was done by DinoBektesevic

Reply:
Uh oh, I cloned the current tagged release on GitHub: https://github.com/fracdiff/fracdiff/commit/93ce110a4b8e7022d3507b83eb64bef4fa685179 which is marked as 0.9.1 - is this a problem?